core/src/test/java/kafka/test/ClusterInstance.java

```diff
@@ -115,6 +116,8 @@ default Optional<ListenerName> controlPlaneListenerName() {
      */
     Map<Integer, BrokerFeatures> brokerFeatures();
 
+    Collection<ControllerServer> controllerServers();
```

Comment:
Perhaps it would be better not to expose this here if it is not supported by both implementations. We can still cast the `ClusterInstance` to `RaftClusterInstance` to get access to the method.